app/javascript/components/physical-storage-form/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Unfortunately, miq-bot doesn't do linting on javascript files. Can you please manually run `eslint` on these files? It should be available through `yarn lint` or you can integrate it to your editor...